controller/ProviderAdmin.php

```diff
@@ -41,4 +44,32 @@ public function getClassService()
     {
         return $this->getServiceLocator()->get(RdfLtiProviderRepository::class);
     }
+
+    protected function getExtraValidationRules(): array
```

Comment:
This is not extra validation rules but lti version validator (from method content)

Reply:
But this is a protected method override, we should not put LTI in the name. This will be used to set custom validators on RDF controllers (check the tao-core PR first). The `Extra` in the name is because it adds validators regardless the existent ones.